cmake/PythonInstallCLITools.py

```diff
@@ -58,7 +58,7 @@
 from sys import argv, executable
 
 # expecting one command line argument - the path to the python_lib folder to place the pip package
-pkgs = {'energyplus_launch': '3.7.2', 'energyplus_transition_tools': '2.1.3'}
+pkgs = {'energyplus_launch': '3.7.4', 'energyplus_transition_tools': '2.1.3'}
```

Comment:
Bumped launch version to 3.7.4 which now has an optional argument just for EnergyPlus!